docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,178 @@ up OpenAI authentication to run the code below.
+            ]
+
+      - The :py:func:`mlflow.evaluate()` will also accepts a list input. One notable requirement is that the each
+        list element i.e. input string needs to be wrapped in another list, so they can be passed as a single prediction request to the model endpoint.
+
+    * - A (nested) list of input strings.
+      - 
+        .. code-block:: python
+
+            [
+                [
+                    {
+                        "messages": [
+                            {"role": "system", "content": "Please answer."},
+                            {"role": "user", "content": "What is MLflow?"},
+                        ],
+                        "max_tokens": 100,
+                    },
+                    # ... more dictionary records
+                ]
+            ]
+
+      - Similar requirements as the above list format apply here as well.
+
+
+
+Passing Inference Parameters
+****************************
+
+You can pass additional inference parameters such as ``max_tokens``, ``temperature``, ``n``, to the model endpoint by setting the ``inference_params`` argument in :py:func:`mlflow.evaluate()`. The ``inference_params`` argument is a dictionary that contains the parameters to be passed to the model endpoint. The specified parameters are used for all the input record in the evaluation dataset.
+
+.. note::
+
+    When your input is a dictionary format tha represents request payload, it can also include the parameters like ``max_tokens``. If there are overlapping parameters in both the ``inference_params`` and the input data, the values in the ``inference_params`` will take precedence.
+
+Examples
+********
+
+**Chat Endpoint hosted by a local** `MLflow Deployments Server <../deployments/index.html>`_
+
+.. code-block:: python
+
+    import mlflow
+    from mlflow.deployments import set_deployments_target
+    import pandas as pd
+
+    # Point the client to the local MLflow Deployments Server
+    set_deployments_target("http://localhost:5000")
+
+    eval_data = pd.DataFrame(
+        {
+            # Input data must be a string column and named "inputs".
+            "inputs": [
+                "What is MLflow?",
+                "What is Spark?",
+            ],
+            # Additional ground truth data for evaluating the answer
+            "ground_truth": [
+                "MLflow is an open-source platform ....",
+                "Apache Spark is an open-source, ...",
+            ],
+        }
+    )
+
+
+    with mlflow.start_run() as run:
+        results = mlflow.evaluate(
+            model="endpoiints:/my-chat-endpoint",
+            data=eval_data,
+            targets="ground_truth",
+            inference_params={"max_tokens": 100, "temperature": 0.0},
+            model_type="question-answering",
+        )
+
+**Completion Endpoint hosted on** `Databricks Foundation Models API and External Models <https://docs.databricks.com/en/machine-learning/model-serving/score-foundation-models.html>`_
```

Comment:
Can we separate the links for foundation models and external models (https://docs.databricks.com/en/generative-ai/external-models/index.html) throughout all the docs & docstrings?